src/Database/Relations/AttachOneOrMany.php

```diff
@@ -185,6 +200,20 @@ public function add(Model $model, $sessionKey = null)
             else {
                 $this->parent->reloadRelations($this->relationName);
             }
+
+            /**
+             * @event model.relation.afterAdd
+             * Called after adding a relation to the model (only for AttachOne/AttachMany relations)
+             *
+             * Example usage:
+             *
+             *     $model->bindEvent('model.relation.afterAdd', function (string $relationName, \October\Rain\Database\Model $relatedModel) use (\October\Rain\Database\Model $model) {
+             *         $relatedClass = get_class($relatedModel);
+             *         traceLog("Relation {$relationName} was added to model {$relatedClass}.");
```

Comment:
Isn't $this->related the FileModeel being attached to $this->parent? In which case it would be `$relatedClass was added as $relationName to get_class($model)`?

Reply:
Indeed.

Reply:
fixed